### PR TITLE
add bangolufsen documentation

### PR DESCRIPTION
**Pull request by mj23000**
## Proposed change
Add documentation for the bangolufsen integration.

PR to add brand icons to core_integrations in brands repo: https://github.com/home-assistant/brands/pull/4605

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93462
- Link to parent pull request in the Brands repository: already added https://github.com/home-assistant/brands/pull/3877 https://github.com/home-assistant/brands/pull/3879
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bangolufsen.markdown [hunk 1]
```diff
@@ -0,0 +1,352 @@
+---
+title: Bang & Olufsen
+description: Instructions on how to integrate Bang & Olufsen devices into Home Assistant.
+ha_category:
+  - Media Player
+  - Multimedia
+ha_release: 2023.6.0
```
Comment (klaasnicolaas) [inline source/_integrations/bangolufsen.markdown]:
```suggestion
ha_release: 2023.8
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by edenhaus — CHANGES_REQUESTED** [review]
Please create a new PR in the brand's repo, where you copy the images from `custom_integrations` to `core_integrations`.

**Comment by mj23000** [comment]
> Please create a new PR in the brand's repo, where you copy the images from `custom_integrations` to `core_integrations`.


I have now done this.
https://github.com/home-assistant/brands/pull/4605

**Comment by jpbede** [comment]
Closing this as it was based of the wrong branch so it has tagged a lot of people.
Please open a new PR based on `next`

**Pull request closed without merging**